tutorials/install-github-actions-runner-mac/index.mdx

```diff
@@ -0,0 +1,107 @@
+---
+meta:
+  title: Installing a Github Actions runner on a Mac mini M1
+  description: How to set up and configure a bare metal Mac mini M1 as a self-hosted runner for Github Actionos
+content:
+  h1: Installing a Github Actions runner on a Mac mini M1
+  paragraph: How to set up and configure a bare metal Mac mini M1 as a self-hosted runner for Github Actionos
+tags: mac m1 github-actions
+categories: 
+  - bare-metal
+  - apple-silicon
+dates:
+  validation: 2023-08-11
+  posted: 2023-08-11
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  You have:
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a Mac mini M1](/bare-metal/apple-silicon/how-to/create-mac-mini-m1/)
+  - You have a Github repository with administrator rights.
+</Message>
+
+## Introduction
+
+GitHub Actions is a CI/CD platform that allows users to automate their software development workflows, connected to a Github organization or repository. While GitHub offers online runners with a pay-as-you-go model, self-hosted runners grant increased control over your CI/CD setup and enhanced customization. This is because you manage the runners yourself, rather than relying on GitHub's infrastructure. This tutorial will show you how to set up, configure, and connect a self-hosted runner on a Mac mini M1 for executing macOS pipelines.
+    <Message type="important">
+      Github [recommends](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#self-hosted-runner-security) to only use self-hosted runners with private repositories for security reasons.
+    </Message>
+
+## Installing the runner
+
+1. Go to [Github](https://github.com/), navigate to the main page of your repository and click **Settings**.
+2. In the left sidebar, click **Actions**, then click **Runners**.
+3. Click **New self-hosted runner**. For the image, select **macOS** image and for the architecture, select **ARM64**.
+4. GitHub will prompt you to execute a series of shell commands to download the runtime and do a basic setup. Take note of those commands, as you will need to execute them on your Mac mini M1.
```

Comment:
```suggestion
4. GitHub will prompt you to execute a series of shell commands to download the runtime and configure a basic setup. Take note of those commands, as you will need to execute them on your Mac mini M1.
```
I think this should be a note and not a proper step. It should explain what exactly happens after you select macOS and ARM64 + what you described